src/Providers/MailerProvider.php

```diff
@@ -26,7 +27,7 @@ public function boot()
         $this->app->singleton('mailer', function (Container $container) {
             $view = $container['view'];
 
-            $mailer = new Mailer($view, $container['swift.mailer'], $container['events']);
+            $mailer = new Mailer('mailer', $view, $container['swift.mailer'], $container['events']);
```

Comment:
method signatue changed in laravel 8 as it supports multiple mailers, each given a name.  We just needed to pass name along.